modules/core/src/test/java/org/apache/ignite/internal/managers/discovery/IgniteTopologyPrintFormatSelfTest.java

```diff
@@ -203,6 +206,30 @@ public void testForceServerAndClientDebugLogs() throws Exception {
         doForceServerAndClientTest(log);
     }
 
+    @Test
+    public void checkMessageOnCoordinatorChangeTest() throws Exception {
+        Ignite server1 = startGrid("server1");
```

Comment:
You can use indices instead of full name

Reply:
Why using indices is better?

Reply:
Actually, do it as you wish. Let it leave as is